tools/FakeItEasy.Build/Program.cs

```diff
@@ -67,12 +67,15 @@
 Target(
     "check-docs-links",
     DependsOn("generate-docs"),
-    () => Run("uv", "run linkchecker --ignore-url=sitemap.xml --ignore-url=404.html --check-extern -F html/utf-8/artifacts/docs-link-check.html ./artifacts/docs/index.html"));
+    () => Run(
+        "uv",
+        "run linkchecker --ignore-url=sitemap.xml --ignore-url=404.html --check-extern --user-agent=FakeItEasyDocs/1.0 -F html/utf-8/../artifacts/docs-link-check.html ../artifacts/docs/index.html",

```

Comment:
Had to set a dummy user agent, because some Github URLs like https://github.com/FakeItEasy/FakeItEasy/labels/P1 return a 404 with the default one (LinkChecker/X.Y)

Reply:
I'd enjoy this as a comment in the code, if you're not opposed.

Reply:
Good point

Reply:
Done